src/Control/Director.php

```diff
@@ -649,7 +649,7 @@ public static function baseURL()
         // Check if BASE_SCRIPT_URL is defined
         // e.g. `index.php/`
         if (defined('BASE_SCRIPT_URL')) {
-            return $baseURL . BASE_SCRIPT_URL;
+            return rtrim($baseURL . BASE_SCRIPT_URL, '/') . '/';
```

Comment:
This is technically not related, but matches how the other two scenarios in this method work. This method should always return a value with a trailing slash.